src/features/tasks/models/TaskModel.ts

```diff
@@ -0,0 +1,25 @@
+import Environment from 'core/env/Environment';
+import { IFuture } from 'core/caching/futures';
+import { ModelBase } from 'core/models';
+import TasksRepo from '../repos/TasksRepo';
+import { TaskStats } from '../rpc/getTaskStats';
+import { ZetkinTask } from '../components/types';
+
+export default class TaskModel extends ModelBase {
+  private _orgId: number;
+  private _repo: TasksRepo;
+
+  constructor(env: Environment, orgId: number) {
+    super();
+    this._orgId = orgId;
+    this._repo = new TasksRepo(env);
+  }
+
+  getTask(taskId: number): IFuture<ZetkinTask> {
```

Comment:
I would move the `taskId` to the constructor, so that these methods don't need to accept it.